rfm_dashboard/__init__.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask 
+from flask import render_template
```

Comment:
We can import multiple function in one line by using `,` like `from flask import Flask, render_template`